repostatus.py

```diff
@@ -0,0 +1,37 @@
+import os
+import pandas as pd
+from dotenv import load_dotenv
+import json
+
+# Load environment variables
+load_dotenv(r'C:\Users\home\tf2.4\TF2.4_IVIM-MRI_CodeCollection\local.env')
```

Comment:
We should be able to get around hard coding paths. This won't work for me, for example.